src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -236,7 +236,10 @@ def _transform(self, inpt: Video, params: dict) -> tv_tensors.Video:
         start_index = 0
         frame_inds = np.concatenate(frame_inds) + start_index
 
-        outputs = torch.stack([torch.tensor(inpt[idx].data) for idx in frame_inds], dim=0)
+        outputs = torch.stack(
+            [torch.tensor(cv2.cvtColor(inpt[idx].data, cv2.COLOR_RGB2BGR)) for idx in frame_inds],
```

Comment:
It seems that this code always converts color format. Could you change a code to convert only if it's `image_color_channel` is RGB? You can check this by adding `_check_inputs` function to `DecodeVideo` class.

Reply:
image_color_channel is not counted in action task since it does not consider video case. So firstly we need to sync default channels of torch and ov inference. And then secondly we need to implement the way to apply image_color_channel for the video case.
This PR aims the first step, and we need additional PR for the second step

Reply:
@jaegukhyun I added error when user change ImageColorChannel value to RGB. Currently action classification task should use BGR format.

Reply:
I don't understand why this part is needed if `ImageColorChannel=BGR` is forced.
Because `cv2.COLOR_RGB2BGR` and `cv2.COLOR_BGR2RGB` are actually the same operation.
If BGR image is given, this part will convert the image to RGB.
https://docs.opencv.org/4.x/d8/d01/group__imgproc__color__conversions.html
![image](https://github.com/openvinotoolkit/training_extensions/assets/97221135/27c8bb5b-527f-4af1-b625-11ab498fc735)

Reply:
Video from datumaro always RGB format. However Video from mmaction always BGR format. So this PR aims to change the color format of video from datumaro to BGR format

Reply:
Yes, as Jaeguk said, current action classification task doesn't support ImageColorChannel. So user should use BGR format regardless of `ImageColorChannel` value. To avoid confusion, this code notifies user that it's not possible to set `ImageColorChannel`.

Reply:
Thanks for clarification.